src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -155,14 +152,11 @@ object OverwatchScope extends Enumeration {
   // Todo Issue_77
 }
 
-object Layer extends Enumeration {
-  type Layer = Value
-  val bronze, silver, gold, consumption = Value
-}
-
-object Frequency extends Enumeration {
-  type Frequency = Value
-  val milliSecond, daily = Value
+object WriteMode extends Enumeration {
+  type WriteMode = Value
+  val append: Value = Value("append")
+  val overwrite: Value = Value("overwrite")
+  val merge: Value = Value("merge")
```

Comment:
Oh I think you're saying WriteMode "merge" wordchoice is bad. I get what you're saying I think. This is 100% an Overwatch term but it could be confusing for devs. Do you have alternative word that sounds better. Happy to adjust, will do in subsequent PR if we find a better word.